### DISCUSSION TITLE
断电后打开软件提示database disk image is malformed

### DISCUSSION THREAD

**Discussion by zhdabiao** [Chia-Network/chia-blockchain#8814] [Support]
无法获取同步!!!!!database disk image is malformed

**Comment by BrandtH22** [comment]
@qswszwj , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note with a corrupt database you will need to delete the database from `~\.chia\mainnet\db` and either sync from genesis or use the official torrent snapshot:
1 .Download the torrent file from https://www.chia.net/downloads/#database-checkpoint
2. Use a torrent client to download the full db
3. Unpack/reassemble the torrent file that was downloaded (on windows I used 7zip, Mac and linux have built in tools that work for this)
4. Move the db to the correct folder (~\.chia\mainnet\db\) and update the name to "blockchain_v2_mainnet.sqlite" (removing the date information in the name)
5. Verify the config file (~\.chia\mainnet\config\config.yaml) has the correct value under the full_node section for `database_path: db/blockchain_v2_CHALLENGE.sqlite` (should only need to change the v1 to v2)
6. Launch chia and wait for a bit (the height to hash and peers files need to be built so this can take 5-10 minutes)

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
